db/migrate/20210304013614_insert_initial_genres.rb

```diff
@@ -0,0 +1,38 @@
+class InsertInitialGenres < ActiveRecord::Migration[6.1]
+  def change
```

Comment:
この操作はロールバックできないので， `up` にしておきましょう。

```suggestion
  def up
```